vervet-underground/internal/scraper/s3_scraper_test.go

```diff
@@ -0,0 +1,180 @@
+package scraper_test
+
+import (
+	"context"
+	"net/http/httptest"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/rs/zerolog/log"
+
+	"vervet-underground/config"
+	"vervet-underground/internal/scraper"
+	"vervet-underground/internal/storage/s3"
+)
+
+const (
+	localstackAccessKey  = "test"
+	localstackSecretKey  = "test"
+	localstackSessionKey = "test"
+	awsEndpoint          = "http://localhost:4566"
+	awsRegion            = "us-east-1"
+)
+
+var s3Cfg = &s3.Config{
+	AwsRegion:   awsRegion,
+	AwsEndpoint: awsEndpoint,
+	Credentials: s3.StaticKeyCredentials{
+		AccessKey:  localstackAccessKey,
+		SecretKey:  localstackSecretKey,
+		SessionKey: localstackSessionKey,
+	},
+}
+
+func isCIEnabled() bool {
+	ci, err := strconv.ParseBool(os.Getenv("CI"))
+	return err == nil || ci
+}
+
+func cleanup() {
+	// cleanup
+	st := s3.New(s3Cfg)
+	revs, err := st.ListObjects("", "")
+
+	if err != nil {
+		log.Warn().Err(err).Msg("failed to List Objects")
+	}
+	for _, rev := range revs.Contents {
+		err = st.DeleteObject(*rev.Key)
+		if err != nil {
+			log.Warn().Err(err).Msg("failed to delete Object")
+		}
+	}
+}
+
+func TestS3Scraper(t *testing.T) {
+	c := qt.New(t)
+	if isCIEnabled() {
+		c.Assert(true, qt.IsTrue)
+		return
```

Comment:
```suggestion
		c.Skip("CI not enabled")
```

https://pkg.go.dev/testing#T.Skip doesn't need the early return -- it will end the test for you.